server/server.go

```diff
@@ -241,6 +241,8 @@ func (s *serve) getAlerts(req *http.Request) []prometheus.Alert {
 		alertName := req.URL.Query().Get(fmt.Sprintf("alertName.%d", i))
 		annotations := s.getMapFromString(req.URL.Query().Get(fmt.Sprintf("alertAnnotations.%d", i)))
 		labels := s.getMapFromString(req.URL.Query().Get(fmt.Sprintf("alertLabels.%d", i)))
+		persistant := req.URL.Query().Get(fmt.Sprintf("alertPersistent.%d", i)) == "true"
```

Comment:
not a big deal functionally, but there is a typo here:

`persistant` should be `persistent`

Reply:
Thanks! PR in https://github.com/vfarcic/docker-flow-monitor/pull/37